nilearn/image/image.py

```diff
@@ -665,3 +665,38 @@ def threshold_img(img, threshold, mask_img=None):
     threshold_img = new_img_like(img, img_data, affine)
 
     return threshold_img
+
+
+def math_img(formula, **imgs):
+    """Interpret a string formula using niimg in named parameters.
+
+    Parameters
+    ----------
+    formula: string
+        The mathematical formula to apply to image internal data.
+
```

Comment:
Can you detail the "**imgs" argument, and well as add a "returns" description.

It would also be good to have an "example" section, that shows couple of example of usage (see the docstring of any numpy function).

You can have a look at https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections and 
http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_numpy.html